docs/Model_Assumptions.rst

```diff
@@ -2,6 +2,27 @@
 Modeling Assumptions of the MVS
 ================================
 
+Cost calculations
+-----------------
+
+The optimization in the MVS is mainly a cost optimization. There are some additional constraints that can be introduced, mainly by adding bounds eg. by limiting the maximum capacity that can be installed (comp. :ref:`maxcap-label`) or adding constraints for certain key performance indicators (see :ref:`constraints-label`). To optimize the energy systems properly, the economic data provided with the input data has to be pre-processed (also see :ref:`economic_precalculation-label`) and then also post-processed when evaluating the results. Following assumptions are important:
+
+* **Project lifetime**: The simulation has a defined project lifetime, for which continuous operation is assumed - which means that the first year of operation is exactly like the last year of operation. Existing and optimized assets have to be replaced to make this possible.
+* **Simulation duration**: It is advisable to simulate whole year to find the most suitable combination of energy assets for your system. Sometimes however you might want to look at specific seasons to see their effect - this is possible in the MVS by choosing a specific start date and simulation duration.
+* **Asset costs**: Each asset can have development costs, specific investment costs, specific operation and management costs as well as dispatch costs.
+    * *Replacement costs* are calculated based on the lifetime of the assets, and residual values are paid at the end of the project.
+    * *Development costs* are costs that will occurr regardless of the installed capacity of an asset - even if it is not installed at all. It stands for system planning and licensing costs. If you have optimized your energy system and see that an asset might not be favourable (zero optimized capacities), you might want to run the simulation again and remove the asset, or remove the development costs of the asset.
```

Comment:
I think a log message (e.g. warning) would be nice in this case.

Reply:
Created issue.